core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2114,6 +2114,8 @@ class SparkContext(config: SparkConf) extends Logging {
    * @param exitCode Specified exit code that will passed to scheduler backend in client mode.
    */
   def stop(exitCode: Int): Unit = {
+    logInfo(s"OS info ${System.getProperty("os.name")}, ${System.getProperty("os.version")}, " +
+      s"${System.getProperty("os.arch")}, Java version ${System.getProperty("java.version")}")
```

Comment:
Hmm.. I would expect to see these when we start it (not stop).

Reply:
Thanks for the review comment. I fixed the comment